lisa/sut_orchestrator/azure/common.py

```diff
@@ -330,7 +357,8 @@ def shared_gallery(self) -> Optional[SharedImageGallerySchema]:
             # The lower() normalizes the image names,
             #  it has no impact on deployment.
             self.shared_gallery_raw = dict(
-                (k, v.lower()) for k, v in self.shared_gallery_raw.items()
+                (k, v.lower()) if k != "network_data_path" else (k, v)
```

Comment:
If it's needed, it should give names of shared_gallery_image, instead of `network_data_path`. We may add more fields in future, so it shouldn't a place, which we have to update every time.

Reply:
Done.
Added "image_name_keys" in AzureVmMarketplaceSchema and SharedImageGallerySchema.